cmd/realm/interfaces.go

```diff
@@ -333,6 +349,9 @@ func getInterfaceDefinition(realm, interfaceName string, interfaceMajor int) (in
 		return interfaces.AstarteInterface{}, err
 	}
 
+	// don't care about `interface sync` because its flag is always false
```

Comment:
Is this comment relevant? (check also the ones below)

Reply:
It is relevant, but a bit too obscure. Will reword.